src/hooks/useAccount.ts

```diff
@@ -4,6 +4,8 @@ import { useStore } from 'src/store';
 import { computed, ref, watch } from 'vue';
 import { SubstrateAccount } from 'src/store/general/state';
 
+export const EthereumExtension = 'Ethereum Extension';
```

Comment:
Just tiny thing but I'm confused a bit it's variable. maybe if it's constant value, uppercase might be better?

Reply:
@sirius651 Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/674/commits/24e171d94a7ef9e40cbbd0ec92424af81bcc4301) commit.